src/flag_gems/utils/pointwise_dynamic.py

```diff
@@ -316,7 +328,7 @@ def generate_destination_passing_pointwise_wrapper(
 
     # task partitioning, 1d task indexing
     tile_size = 512
-    num_warps = 4
+    num_warps = 1
```

Comment:
why?

Reply:
for our mlu, pointwise op num_warps = 1 's perf is better